.github/workflows/ci-backend.yml

```diff
@@ -31,14 +31,6 @@ jobs:
         export LD_LIBRARY_PATH=${jep_path}:${jep_site_packages_path}:${jep_lib_path}:${LD_LIBRARY_PATH}
         export LD_PRELOAD=${jep_lib_path}/libpython3.so
 
-        echo "Downloading Spark 2.1.1"
-        wget -q https://archive.apache.org/dist/spark/spark-2.1.1/spark-2.1.1-bin-hadoop2.7.tgz
-        tar xfz spark-2.1.1-bin-hadoop2.7.tgz
-        rm spark-2.1.1-bin-hadoop2.7.tgz
-        export SPARK_HOME=`pwd`"/spark-2.1.1-bin-hadoop2.7"
-        export PATH="$SPARK_HOME/bin:$PATH"
-
-        echo "Testing with Scala 2.11"
         pushd $GITHUB_WORKSPACE
-          sbt ++2.11.12 test
-        popd
+          sbt +test
```

Comment:
FWIW @jonathanindig, sbt now does the CI for all versions. That's why this junk was removed.

Additionally, I bumped up the Spark version under test to the latest Spark 3 version, because that version by default is built with Scala 2.12 so there's no shenanigans that have to be done to get the 2.12 version, and then install Hadoop, and etc.